miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -291,7 +298,7 @@ private static DetailedSampleDto asDetailedSampleDto(DetailedSample from) {
     return dto;
   }
 
-  private static DetailedSample toDetailedSample(DetailedSampleDto from) {
+  private static DetailedSample toDetailedSample(DetailedSampleDto from, SampleClassCategorizerer categorizer) throws IOException {
```

Comment:
The Service layer is responsible for looking up DB data. Dto conversion should not do this